agent/util-scripts/pbench-metadata-log

```diff
@@ -151,6 +151,14 @@ function metadata_log_start {
     local dir=$1
     local group=$2
     local ts=$3
+
+    # Opportunistically capture the caller's SSH configuration file in
+    # case they have host specific configurations they might want to
+    # consider in the future when reviewing historical data.  The host
+    # names used in the config file might be very different from what
+    # tools capture in their output.
+    cp ${HOME}/.ssh/config ${dir}/ssh.config > /dev/null 2>&1
```

Comment:
While we're at it, perhaps also capture /etc/ssh/ssh_config and /etc/ssh/ssh_config.d/*?

Reply:
Done.